channels/alexa/src/main/kotlin/com/justai/jaicf/channel/alexa/AlexaReactions.kt

```diff
@@ -97,11 +98,14 @@ class AlexaReactions(
         response.builder.addAudioPlayerStopDirective()
     }
 
-    fun endSession(text: String? = null) {
-        text?.let { say(it) }
+    fun endSession(text: String): EndSessionReaction {
```

Comment:
Why did you change the method's signature? `text` arg should be nullable for multi-channel scenarios where some content could be generated for many channels before the session ending.

Reply:
Ok